src/lib/rng/jitter_rng/jitter_rng.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+* CPU Jitter Random Number Generator
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/jitter_rng.h>
+
+extern "C" {
```

Comment:
The version of the header on my system already includes `extern "C"` blocks if `__cplusplus` is defined; is this really necessary with any (recent) version?

Reply:
Done.